include/ignition/fuel_tools/CollectionIdentifier.hh

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (C) 2020 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#ifndef IGNITION_FUEL_TOOLS_COLLECTIONIDENTIFIER_HH_
+#define IGNITION_FUEL_TOOLS_COLLECTIONIDENTIFIER_HH_
+
+#include <memory>
+#include <string>
+
+#include "ignition/fuel_tools/Helpers.hh"
+
+namespace ignition
+{
+  namespace fuel_tools
+  {
+    /// \brief Forward Declaration
+    class CollectionIdentifierPrivate;
+    class ServerConfig;
+
+    /// \brief Defines how to identify a world
```

Comment:
I think this documentation needs to be fixed.

Reply:
There are other instances where `world` is used in the documentation.

Reply:
https://github.com/ignitionrobotics/ign-fuel-tools/pull/98/commits/22c730bf12b0aa468d61f9477836b36405f542ab